docs/getting-started.md

```diff
@@ -6,3 +6,81 @@ description: Getting started with pass4s
 # Getting started
 
 This documentation is work in progress
+
+
+# Getting started
+
+## About
+
+Pass4s is a Scala library providing an abstract layer for cross app messaging. It provides implementations for platforms like AWS SQS/SNS and ActiveMQ.
+
+## Basis installation
+
+The library is divided into multiple modules. If you're only interested in the basic abstractions, add following to your `build.sbt` file:
+
+```scala
+// Algebraic abstractions (Sender/Consumer)
+"com.ocadotechnology" %% "pass4s-kernel" % "@VERSION@"
+
+// Message, Destination, CommittableMessage, Connector
+"com.ocadotechnology" %% "pass4s-core" % "@VERSION@"
+
+// Broker
+"com.ocadotechnology" %% "pass4s-high" % "@VERSION@"
+```
+
+## Modules 
+
+### Connectors
+
+ActiveMq
```

Comment:
maybe add links to explain what activemq and others are, like https://activemq.apache.org/